data/json/recipes/electronic/components.json

```
@@ -15,6 +15,122 @@
+    "time": "300 m",
+    "reversible": true,
+    "decomp_learn": 4,
+    "book_learn": [ [ "textbook_fabrication", 4 ], [ "manual_electronics", 4 ], [ "mag_electronics", 4 ], [ "manual_mechanics", 4 ] ],
+    "using": [ [ "soldering_standard", 50 ], [ "steel_standard", 15 ] ],
+    "qualities": [
+      { "id": "HAMMER", "level": 1 },
+      { "id": "SAW_M_FINE", "level": 1 },
+      { "id": "SAW_M", "level": 1 },
+      { "id": "SCREW", "level": 1 }
+    ],
+    "components": [
+      [ [ "e_scrap", 5 ] ],
+      [ [ "power_supply", 2 ] ],
+      [ [ "amplifier", 4 ] ],
+      [ [ "bearing", 40 ] ],
+      [ [ "cable", 30 ] ],
+      [ [ "motor_oil", 500 ] ]
+    ]
+  },
+  {
+    "type": "recipe",
+    "result": "motor_enhanced",
+    "category": "CC_ELECTRONIC",
+    "subcategory": "CSC_ELECTRONIC_PARTS",
+    "skill_used": "electronics",
+    "difficulty": 6,
+    "skills_required": [ [ "fabrication", 4 ], [ "mechanics", 5 ] ],
+    "time": "480 m",
+    "reversible": true,
+    "decomp_learn": 6,
+    "book_learn": [ [ "textbook_robots", 5 ], [ "recipe_lab_elec", 5 ] ],
+    "using": [ [ "soldering_standard", 75 ], [ "steel_standard", 20 ], [ "welding_standard", 10 ] ],
+    "qualities": [
+      { "id": "HAMMER", "level": 3 },
+      { "id": "SAW_M_FINE", "level": 1 },
+      { "id": "SCREW_FINE", "level": 1 },
+      { "id": "WRENCH_FINE", "level": 1 },
+      { "id": "SAW_M", "level": 1 },
+      { "id": "SCREW", "level": 1 }
+    ],
+    "components": [
+      [ [ "e_scrap", 10 ] ],
+      [ [ "circuit", 4 ] ],
+      [ [ "processor", 1 ] ],
+      [ [ "power_supply", 3 ] ],
+      [ [ "amplifier", 4 ] ],
+      [ [ "bearing", 50 ] ],
+      [ [ "cable", 90 ], [ "carbon_fiber_filament", 30 ] ],
+      [ [ "motor_oil", 750 ] ],
+      [ [ "alloy_sheet", 10 ] ]
+    ]
+  },
+  {
+    "type": "recipe",
+    "result": "motor_super",
+    "category": "CC_ELECTRONIC",
+    "subcategory": "CSC_ELECTRONIC_PARTS",
+    "skill_used": "electronics",
+    "difficulty": 8,
+    "skills_required": [ [ "fabrication", 6 ], [ "mechanics", 7 ] ],
+    "time": "12 h",
+    "reversible": true,
+    "decomp_learn": 8,
+    "book_learn": [ [ "recipe_lab_elec", 8 ] ],
+    "using": [ [ "soldering_standard", 100 ], [ "steel_standard", 60 ], [ "welding_standard", 10 ] ],
+    "qualities": [
+      { "id": "HAMMER", "level": 3 },
+      { "id": "SAW_M_FINE", "level": 1 },
+      { "id": "SAW_M", "level": 1 },
+      { "id": "SCREW_FINE", "level": 1 },
+      { "id": "WRENCH_FINE", "level": 1 },
+      { "id": "SCREW", "level": 1 }
+    ],
+    "components": [
+      [ [ "e_scrap", 20 ] ],
+      [ [ "circuit", 6 ] ],
+      [ [ "processor", 2 ] ],
+      [ [ "self_monitoring_module", 1 ] ],
+      [ [ "power_supply", 4 ] ],
+      [ [ "amplifier", 6 ] ],
+      [ [ "bearing", 60 ] ],
+      [ [ "motor_oil", 1000 ] ],
+      [ [ "carbon_fiber_filament", 75 ] ],
+      [ [ "alloy_sheet", 15 ] ]
```

This one isn't superalloy, the robotics components and carbon fiber already seems reasonable a component requirement.
```suggestion
      [ [ "carbon_fiber_filament", 75 ] ]
```